lib/utils/javascript.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+  duplicateArrays,
```

Comment:
Can we only copy `flatMap` and `flat` (rename `flatten` to `flat`) for now since those are basically just polyfills of native JavaScript functions? The other utils are custom and would just become deadcode anyway.